subsys/partition_manager/pm.yml.pcd

```diff
@@ -1,8 +1,13 @@
 #include <autoconf.h>
-
 # This block of RAM is used for communicating Network Core firmware update
 # metadata
 pcd_sram:
-  placement: {before: [rpmsg_nrf53_sram, end]}
+  placement: {after: [end]}
```

Comment:
As long as it's not the root node it is allowed. however I don't think it has the intended purpose
https://yaml.org/spec/1.2-old/spec.html#id2764044 

The idea was that placement would be merged with multiple keys however it seems that the first node overrides every other. I'll change it back to have `end` on both placements.

Reply:
It would resolve to 
```yaml
pcd_sram: 
  placement: 
    after: 
      - rpmsg_nrf53_sram
```
So I'll put the end back in both places, was just trying to save some copied configuration.

Reply:
Out of curiosity, what exactly `after: end` means? I'm not that familiar with inner workings of the PM and it seems a bit odd from the linguistic point of view ;)

Reply:
It sounds a bit weird I agree. What it means that it will try to place it after the end of ram which it obviously cannot do so it will place it near end of ram. Maybe I should change it to `after: rpmsg, before: end` if it's possible.

Reply:
Decided to put it `after: start` so that it will always be at the start of RAM. So if it's secure/non-secure split it will be at the start of the secure RAM and if it's just secure(trustzone not enabled). It's placed in the start of RAM before `sram_primary`